debug-zip.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Collect Debug & Error Logs from Your Cluster
+summary: Learn the commands for configuring, starting, and managing a CockroachDB cluster.
+toc: false
+---
+
+The `debug zip` [command](cockroach-commands.html) connects to your cluster and gathers [log files](debug-and-error-logs.html) from each active node into a single file (inactive nodes are not included).
```

Comment:
`debug zip` collects much more information than just the log files.
It also includes:
- schema change events
- node liveness
- gossip data
- stack traces
- range lists
- list of databases and tables